images/build-e2e/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.16.7 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as builder
```

Comment:
Only go-toolset:1.17.7 is available.

Reply:
Yeah it is again https://github.com/code-ready/crc/blob/main/update-go-version.sh#L15-L16, looks like after executing this script we have to manually check or if there is way to get latest toolset version from registry.

Reply:
Yeah, it had the same issue with 1.16, the version needed to be manually adjusted.

Reply:
@cfergeau Updated.